src/main/resources/i18n/messages.properties

```diff
@@ -766,12 +767,16 @@ replay.replayDurationTooltip = Replay length
 replay.playerCountTooltip = Players
 replay.ratingTooltip = Average rating
 replay.qualityTooltip = Quality
+replay.deleteTooltip = Delete local replay file
+replay.delete = Delete Replay
+replay.deleteLocal = Delete local replay
```

Comment:
why delete and deleteLocal I would think we only need one

Reply:
`replay.delete` is used as a label for the Delete Replay button, while `replay.deleteLocal` is used as part of the notification title. To me, the difference in capitalization and verbosity make sense in each of their context. Let me know if we need to minimize the number of translations.